### PR TITLE
Add test pipeline segment

### PR DESCRIPTION
**Pull request by lebovic**
Adds a `test` pipeline segment.

Adds:
- `test` pipeline segment to allow demonstrating/trying out the interface without a specific tool to distract.
- `.env` handling with `python-dotenv`

### CODE REVIEW

File: toolchest_client/query.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class Query():
     """
 
     # Base URLs used by the server API.
-    BASE_URL = "https://api.toolche.st"
+    BASE_URL = os.environ.get("BASE_URL", "https://api.toolche.st")
```
Comment (lebovic) [inline toolchest_client/query.py:28]: Note that this isn't setting a `DEPLOY_ENVIRONMENT` environment variable.

To set, you can create a `.env` file in the root of the project.

File: toolchest_client/tools.py [hunk 2]
```diff
@@ -120,3 +122,31 @@ def kraken2(tool_args="", **kwargs):
         output_name="output.txt",
         **kwargs
     )
+
+
+def test(tool_args="", **kwargs):
+    """Run a test pipeline segment via Toolchest. A plain text file containing 'step one' is returned."
```
Comment (bcai2) [inline toolchest_client/tools.py:128]: Is the last quote a typo? Otherwise, everything else looks good.

**Review by bcai2 — APPROVED** [review]
LGTM, see comment about a typo

File: toolchest_client/tools.py (hunk 2 above)
Comment (lebovic) [reply toolchest_client/tools.py:128]: Whoops, yep. It's `success` now.

**Pull request merged**